pkg/scannerv4/client/client.go

```diff
@@ -107,7 +107,11 @@ func createGRPCConn(ctx context.Context, o connOptions) (*grpc.ClientConn, error
 // GetIndexReport, then if not found or if the report is not successful, then
 // call CreateIndexReport.
 func (c *gRPCScanner) GetOrCreateImageIndex(ctx context.Context, ref name.Digest, auth authn.Authenticator) (*v4.IndexReport, error) {
-	ctx = zlog.ContextWithValues(ctx, "component", "scanner/client", "method", "GetOrCreateImageIndex")
+	ctx = zlog.ContextWithValues(ctx,
```

Comment:
this doesn't change the logs, but it does add more context, if logs are added in the future